### ISSUE TITLE
Chapter 5

### ISSUE THREAD

**Issue by cschaffner** [BasicProbability/LectureNotes#30]
I've left a few more comments in Chapter 5, and I changed some little things. @philschulz can you have a look?

**Comment by cschaffner** [comment]
Hey @philschulz , I think it's high time to finalize chapter 5! Can you take care of the few comments from a2def7b98f1a14fdc4dd8d65dcc32f6a9db6dbb0 I left and incorporate my changes into main, and add them to the full script? Also, there was this issue of Markov's inequality holding only for non-negative RVs and positive a.

**Comment by philschulz** [comment]
Hey Christian, 

thanks for fixing up the proof of the factorisation theorem! I addressed your second comments (my concern there was that the interval is closed and thus the derivative at the boundary points does not exist -- I now changed the formulation). As for your first point, I don't think that providing the data will be helpful. The whole point is that bigger samples lead to better estimates. What these samples actually are is of little interest. In fact, when I generated these plots I never even saw the random data myself. Everytime I regenerate these plots, they'll look slightly different. Instead of giving the data, I expanded the explanation in the caption somewhat.
